packages/react-dom/src/__tests__/ReactDOMServerIntegrationFragment-test.js

```diff
@@ -103,7 +103,15 @@ describe('ReactDOMServerIntegration', () => {
     });
 
     itRenders('an empty fragment', async render => {
-      expect(await render(<React.Fragment />)).toBe(null);
+      expect(
+        (
+          await render(
+            <div>
+              <React.Fragment />
+            </div>,
+          )
+        ).firstChild,
+      ).toBe(null);
```

Comment:
Why is this null? Seems unexpected.

Reply:
It's the child of the container we're asserting on. That's what the original test was really testing that a fragment doesn't add anything.

Reply:
D'oh. I didn't notice `.firstChild`. My bad.